### DISCUSSION TITLE
HTML Truncation Flawed

### DISCUSSION THREAD

**Discussion by brimwats1** [wangchucheng/hugo-eureka#117] [Q&A]
**Describe the bug**

Truncated summaries of pages improperly rendering HTML

**To Reproduce**

Create a page. In the YAML or first lines of post include apostrophes or ampersands.

Generate hugo.

**Expected behaviour**

Apostrophes and ampersands rendered correctly

**Screenshots**

You can see the issue here: https://ncpwg.org/resources/
![image](https://user-images.githubusercontent.com/49103046/119776533-ea877c00-be79-11eb-86da-5861a799aba8.png)

When the page is clicked into it's fine:

![image](https://user-images.githubusercontent.com/49103046/119776595-f96e2e80-be79-11eb-846a-274a674d8b50.png)

**Comment by wangchucheng** [comment]
Hi,

I just tested `we&rsquo;ve` in my environment, and it seems great:
![image](https://user-images.githubusercontent.com/38368052/120312102-e1c7f780-c30a-11eb-8c58-978a08a80dbd.png)

If there is anything I can help you, please feel free to tell me.

**Comment by brimwats1** [comment]
hmmm! any idea why it does it for me? I haven't changed any eureka settings: 
![image](https://user-images.githubusercontent.com/49103046/120329844-1a32eb80-c2a1-11eb-83ec-8555ee9f724d.png)
![image](https://user-images.githubusercontent.com/49103046/120329854-1bfcaf00-c2a1-11eb-8299-b0d6e6329baa.png)

link
https://ncpwg.org/resources/
https://ncpwg.org/resources/resource-press/

**Comment by wangchucheng** [comment]
Well, this seems to be an unreproducible problem, but you can share the repo with us if you want. Let us see what we can do.

**Comment by brimwats1** [comment]
Absolutely! How would you recommend I do that? I could add you to the repo, or zip it and share.

**Comment by wangchucheng** [comment]
You can just give me the url of your repo if it is public.


________________________________
发件人: Brian M. Watson ***@***.***>
发送时间: Thursday, June 3, 2021 2:36:14 AM
收件人: wangchucheng/hugo-eureka ***@***.***>
抄送: C. Wang ***@***.***>; Comment ***@***.***>
主题: Re: [wangchucheng/hugo-eureka] HTML Truncation Flawed (#108)


Absolutely! How would you recommend I do that? I could add you to the repo, or zip it and share.

―

**Comment by brimwats1** [comment]
It's not currently, because my personal email is in the formspree field, which is why i offered the other ones :)

**Comment by wangchucheng** [comment]
Well, you can zip it and share with us. And if time permits, we may be able to see what happened to your project.


________________________________
发件人: Brian M. Watson ***@***.***>
发送时间: Thursday, June 3, 2021 11:51:42 PM
收件人: wangchucheng/hugo-eureka ***@***.***>
抄送: C. Wang ***@***.***>; Comment ***@***.***>
主题: Re: [wangchucheng/hugo-eureka] HTML Truncation Flawed (#108)


It's not currently, because my personal email is in the formspree field, whichis why i offered the other ones

―

**Comment by brimwats1** [comment]
Thanks so much @wangchucheng ! It is attached when you have the time — Let me know when you've downloaded it and I can delete the zip file here :)

**Comment by wangchucheng** [comment]
I have downloaded the file, but I am afraid it will take a few days before I can check it.

**Comment by brimwats1** [comment]
totally fine and understand! thanks for your labor!

**Comment by wangchucheng** [comment]
Hi,

I was trying to check it today, but it seems that this zip is empty. Did an error occur while uploading the file?

Btw, this issue seems to be unreproducible, so I will convert it to a discussion. Because discussion is a more suitable place to ask for support and I will continue to help you with this.
